packages/edit-site/package.json

```diff
@@ -32,7 +32,9 @@
 		"@wordpress/hooks": "file:../hooks",
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/media-utils": "file:../media-utils",
-		"@wordpress/notices": "file:../notices"
+		"@wordpress/notices": "file:../notices",
+		"file-saver": "^2.0.2",
+		"jszip": "^3.2.2"
```

Comment:
Is this even used? It drew my attention because it's a [26kb+ dependency](https://bundlephobia.com/result?p=jszip@3.2.2), but I don't see that we import it anywhere.

Reply:
It was used to export a theme in a zip file, but that functionality was removed. I think we can keep it as it will soon be introduced again.